examples/drag-n-drop/main.js

```diff
@@ -11,29 +11,39 @@ var most = require('most');
 var DROP = 0, GRAB = 1, DRAG = 2;
 
 module.exports = function() {
+	// The area where we want to do the dragging
+	var area = document.querySelector('.dragging-area');
 	// The thing we want to make draggable
 	var draggable = document.querySelector('.draggable');
+	var dragOffset = {};
```

Comment:
Ah, interesting.  I wonder if there's a way to get the offset right without having to capture and update this as shared mutable state.  For example, could it also pass through the stream, perhaps as a part of the dragInfo object that's already flowing through the stream?